clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/SimpleClusterTierClientEntity.java

```diff
@@ -171,7 +169,7 @@ public boolean isConnected() {
   @Override
   public void validate(ServerStoreConfiguration clientStoreConfiguration) throws ClusterTierException, TimeoutException {
     try {
-      invokeInternalAndWait(endpoint.beginInvoke(), timeouts.getLifecycleOperationTimeout(), messageFactory.validateServerStore(storeIdentifier , clientStoreConfiguration), false);
+      invokeInternalAndWait(endpoint.beginInvoke(), timeouts.getWriteOperationTimeout(), messageFactory.validateServerStore(storeIdentifier , clientStoreConfiguration), false);
```

Comment:
Why is validate a write operation?

Reply:
Something we decided with @ljacomet. What would you use? Connection?

Reply:
Since we added back a connection timeout, I would make validate use that one indeed.

Reply:
Deal. I thought about it but wasn't sure it was making sense. But now I'm convinced :-)